### PR TITLE
New Feature: CORS Access-Control-Allow-Credentials

### PR DESCRIPTION
**Pull request by michaelphipps**
I'm new to contributing so please be gentle if I'm doing it wrong!

I've updated CORS to include an Access-Control-Allow-Credentials header.  
This makes it possible to pass authentication information such as cookies, or in my case use an SSL Client Certificate in order to protect my API.

### CODE REVIEW

**Comment by michaelphipps** [comment]
**I need some help writing tests.**

```
There was 1 failure:

1) Siler\Test\Unit\ResponseTest::testCors
Failed asserting that an array contains 'Access-Control-Allow-Credentials'.

/home/runner/work/siler/siler/tests/Unit/Http/ResponseTest.php:132

```
I have no idea how to write a test for this.

File: tests/Unit/Http/ResponseTest.php [hunk 1]
```diff
@@ -129,6 +129,7 @@ public function testCors()
             $this->assertContains('Access-Control-Allow-Origin: *', $headers);
             $this->assertContains('Access-Control-Allow-Headers: Content-Type', $headers);
             $this->assertContains('Access-Control-Allow-Methods: GET, POST, PUT, DELETE', $headers);
+            $this->assertContains('Access-Control-Allow-Credentials', $headers);
```
Comment (leocavalcante) [inline tests/Unit/Http/ResponseTest.php:132]: I think you forgot to assert the header value (`true`) besides its presence.
```suggestion
            $this->assertContains('Access-Control-Allow-Credentials: true', $headers);
```

Comment (michaelphipps) [reply tests/Unit/Http/ResponseTest.php]: Thank you!  I see it now, lets see how THAT goes!

**Comment by leocavalcante** [comment]
Thanks for your contribution!

**Pull request merged**